app/utils/translations/resources.ts

```diff
@@ -130,6 +130,38 @@ const fr = {
   ...RecurringEmissions.fr,
 };
 
+const cs = {
+  ...UI.cs,
+  ...Calculation.cs,
```

Comment:
`Calculation`, `NoEmission`, `PermissionsRequest` & `OpenFoodFacts` imports are invalid: you forgot to add to do the import in related index files : `import cs from "./cs.json";`

Reply:
ugh.. I'm a little lost. :( unfortunately I'm not very experienced in programming

Reply:
No problem, I took over, thanks :)